CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+# Contributing to 7 days to die sever manager
+You can find all the information about the server [here](https://docs.csmm.app/en/7D2D)
+## Thanks for taking the time to contribute
+The following is a set of guidelines for contributing to 7 days to die sever manager. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+## Links to some resources
+
+You can join the discord server to directly communicate with the developers by clicking [here](http://catalysm.net/discord).
+
+You can find all information about the CSMM [here](https://docs.csmm.app/).
+
+You can find the installation guide [here](https://docs.csmm.app/en/CSMM/self-host/installation.html).
+
+
+
+## Code of Conduct for Contributors
+This project and everyone participating in it is governed by the code of conduct for contributors which you can find [here](CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code.
+
+## How can I contribute?
+1. You can report bugs/glitches/faults that you are likely to come across when using the project server and client apps.
+2. You can suggest enhancements that you would want to see implemented on the project - which gets voted in the roadmap.
+3. You can address a wide variety of open issues if you wish to contribute using your code and make a pull request.
+
+## Steps to contribute
+
+* Comment on the issue you want to work on. Make sure it's not assigned to someone else. For new comers to open-source, you can pick the issue having 'good first issue' tag.
+
+* If you think you encounter a bug or have a suggestion for improvement of code or to add a feature, then create a issue with proper title and tags but first make sure it's not already present. And if you want to work on that, then comment on that issue and wait for the owner to assign that work to you.
+
+### Making a PR
+
+> - Make sure you have been assigned the issue to which you are making a PR.
+> - If you make PR before being assigned, It will be labeled `invalid` and closed without merging.
```

Comment:
> - Make sure you have been assigned the issue to which you are making a PR.
> - If you make PR before being assigned, It will be labeled `invalid` and closed without merging.

absolutely not true. We (as maintainers) make issues for things we want to fix, but welcome changes that people want to implement. There's no need for an issue or for it to be assigned first.

Reply:
Ok, I read that on contributing guide on hacktoberfest. So I thought to add that, But I will remove it.

Reply:
It varies from repo to repo :)